### DISCUSSION TITLE
二级标题到底要不要加粗

### DISCUSSION THREAD

**Discussion by MingHong1019** [HFUTTUG/HFUT_Thesis#41] [Q&A] [answered]
看编译完的示例后，发现pdf的二级标题没有加粗。感觉有点奇怪，于是阅读了《[合肥工业大学研究生学位论文写作规范》，发现这个规范本身就很不规范，比如说：
《规范》定义二级标题：
二级（节）标题：黑体，小四号，左对齐
然而规范本身二级标题却加了粗
《规范》定义正文：
正文：宋体，小四号，左对齐，首行缩进2字符
然而规范正文却是居中对齐。

所以要是想在pdf中加粗二级标题，要修改哪里呢？（这些奇怪的格式问题我还是找个时间问问导员去吧。。）

**Comment by LaureatePoet** [comment, marked as answer]
你的问题可以通过阅读“[CTeX 宏集手册](http://mirrors.ibiblio.org/CTAN/language/chinese/ctex/ctex.pdf)”来系统解决。

----

关于本模板修改章节标题格式。

可以对如下部分进行修改。

https://github.com/HFUTTUG/HFUT_Thesis/blob/6f0efdac293d749dbd65ce069cf97dbe19faebad/hfutthesis.cls#L2472-L2570

----
对于你提出的“二级（节）标题”对应的`section`，修改`format`即可。

即：
```TeX
    section = {
    format     = \sffamily\bfseries\fontsize{12bp}{15.6bp}\selectfont,
    aftername  = \hspace{0.5\ccwd},
    beforeskip = 7.8bp,
    afterskip  = 7.8bp,
  },
```
